extensions/amp-script/0.1/amp-script.js

```diff
@@ -31,6 +32,8 @@ import {isExperimentOn} from '../../../src/experiments';
 /** @const {string} */
 const TAG = 'amp-script';
 
+/** @const {number} */
+const MAX_SCRIPT_SIZE = 150000;
```

Comment:
Nit: add a comment

Reply:
Agreed, the table from the PR description helps readers to understand the limit as chosen.